sdk/test/Services/S3/UnitTests/Custom/ResponseMapperTests.cs

```diff
@@ -319,36 +319,59 @@ private void ValidateMappingTransferUtilityAndSdkRequests<TSourceRequest, TTarge
                 // Resolve alias to actual property name
                 var resolvedPropertyName = ResolvePropertyName(propertyName, sourceType.Name);
                 var sourceProperty = sourceType.GetProperty(resolvedPropertyName);
+                
```

Comment:
added this change because of the issue with contentlanguage. previously we were only setting the property and not the value in the headers. this makes it so we set both the property and the headers value in the source object